.buildkite/scripts/steps/build_api_docs.sh

```diff
@@ -22,11 +22,11 @@ if [[ "${PUBLISH_API_DOCS_CHANGES:-}" == "true" ]]; then
   git add api_docs
   git commit -m "[api-docs] Daily api_docs build"
 
-  git remote add kibanamachine https://github.com/kibanamachine/kibana.git
-  git push -u kibanamachine "$branch"
-  prUrl=$(gh pr create --repo elastic/kibana --title "[api-docs] $(date +%F) Daily api_docs build" --body "Generated by $BUILDKITE_BUILD_URL" --label "release_note:skip" --label "docs")
+  git push origin "$branch"
+
+  prUrl=$(gh pr create --repo elastic/kibana --base main --head "$branch" --title "[test/api-docs] $(date +%F) Daily api_docs build" --body "Generated by $BUILDKITE_BUILD_URL" --label "release_note:skip" --label "docs")
   echo "Opened PR: $prUrl"
-  gh pr merge --repo elastic/kibana --auto --squash "$prUrl"
+  # gh pr merge --repo elastic/kibana --auto --squash "$prUrl"
```

Comment:
Is this WIP? I saw the migration test build failed as well.

Reply:
It's a wip, trying to get that last job working somehow

Reply:
undone!